src/main/webapp/app/overview/course-conversations/course-conversations.component.html

```diff
@@ -32,6 +32,7 @@
                     (onPlusPressed)="onAccordionPlusButtonPressed($event)"
                     [showAddOption]="CHANNEL_TYPE_SHOW_ADD_OPTION"
                     [channelTypeIcon]="CHANNEL_TYPE_ICON"
+                    [sidebarItemShowAlways]="ALWAYS_SHOW"
```

Comment:
Does not necessarily need to be changed, however, I think it would be easier to read and understand if the variable would be named the other way around `SHOW_ALWAYS`.
It is quite cumbersome to read for me like this 😅